### PR TITLE
MNT: Re-rendered with conda-smithy 3.1.6 and pinning 2018.06.19

### PR DESCRIPTION
**Pull request by dnadeau4**

### CODE REVIEW

**Comment by dnadeau4** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,29 +15,22 @@ build:
...
   build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
     - python
-    - hdf5 1.10.1.*
-    - libnetcdf 4.*
+    - libnetcdf
     - lapack >=3.4.2
     - clapack >=3.2.1
```
Comment (isuruf) [inline recipe/meta.yaml]: You can remove `lapack` and `clapack` and replace with openblas.

Comment (dnadeau4) [reply recipe/meta.yaml]: Done

**Comment by dnadeau4** [comment]
@conda-forge-admin, please rerender.

**Pull request merged**